flocker/node/agents/blockdevice.py

```diff
@@ -1505,6 +1511,10 @@ class BlockDeviceDeployer(PClass):
         profiles.
     :ivar FilePath mountroot: The directory where block devices will be
         mounted.
+    :ivar FilePath sharedroot: The parent directory for all dataset paths that
+        will be shared externally. Defaults to being the same as mountroot.
+        This is designed to either be the mountroot or be a directory that is
+        bind mounted read only to the mountroot.
```

Comment:
:question: Not introduced in this patch, but I wish there were an underscore before `root`.